js/sagas/onboarding.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * A saga that manages the Onboarding.
+ *
```

Comment:
I would add a link to the document that describes the onboarding process: https://docs.google.com/document/d/1le-IdjcGWtmfrMzh6d_qTwsnhVNCExbCd6Pt4gX7VGo/edit

Reply:
Added